### ISSUE TITLE
Is something missing from the documentation?

### ISSUE THREAD

**Issue by Birowsky** [NoRedInk/elm-assets-loader#7]
Here's what I have,

in the loader:

        {
          test: /\.elm$/,
          loaders: [
            'elm-assets?module=Assets&tagger=AssetPath',
            'elm-webpack?pathToMake=node_modules/.bin/elm-make'
          ]
        }

the file loader:

        {
          test: /\.(png|jpg|jpeg|eot|svg|ttf|woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$)/,
          loader: ifProd(
            'file-loader?name=[path][name].[hash].[ext]&context=./src',
            'file-loader?name=[path][name].[ext]&context=./src'
          )
        }

and the assets modle:

```
module Assets exposing (AssetPath(AssetPath), towerNarrow, toPath)


type AssetPath
    = AssetPath String


towerNarrow =
    AssetPath "/assets/bitmap/tower-narrow.jpg"


toPath (AssetPath path) =
    path
```

And yet, this is the output, untouched:

```
var _teambirowsky$dscova$Assets$towerNarrow = _teambirowsky$dscova$Assets$AssetPath('tower-narrow.jpg');
```

**Comment by Birowsky** [comment]
I fixed it by providing `package=teambirowsky/dscova`.

Based on the docs:

> Not needed if the tagger is defined in your main application code

this is unnecessary. Or have I missed something?

**Comment by AionDev** [comment]
Related to documentation issue - i wasn't able to use this loader at all[. Asked on stackoverflow here: How to load images used in elm code with webpack 2 ?](http://stackoverflow.com/questions/44077711/how-to-load-images-used-in-elm-code-with-webpack-2/44078966#44078966)

**Comment by ento** [comment]
The readme was wrong; fixed in 222bb95f1efe010a2c9979f018460fa6760a79bd - thanks for issuing, and sorry it took long to fix.

**Comment by ento** [comment]
@Aion-github Tried improving the README [by adding an actual usage example](https://github.com/NoRedInk/elm-assets-loader/compare/222bb95f1efe010a2c9979f018460fa6760a79bd...fb10fee816d7ed190e3a6ceb9d9dc3086de32d47), and posted an answer to the Stack Overflow question. 

Please feel free to open an issue if you find anything else or more things can be clarified.
